code/game/objects/items/weapons/implants/implantfluff.dm

```diff
@@ -0,0 +1,112 @@
+
+/**
+ * # Implant Fluff Datum
+ *
+ * Implant fluff is just lore about the implant that is accessed through the implantpad, you must attach
+ * one of these datums to the implant_data var on an implant for it to show it up.
+ */
+/datum/implant_fluff
+	var/name = "Unknown Implant"
+	var/life = "Not Specified"
+	var/notes = "No Notes"
+	var/function = "An electronic insert that supplements biological functions"
+
+/datum/implant_fluff/abductor
+	name = "GC$X9%D#G"
```

Comment:
Wonder if it'd be possible to make this use wingdings like grey speech?

Reply:
It would be cool, but also I'd have to add in custom TGUI behavior to set the font to wing dings, which is pretty doable but I'm also getting ready to move back in for college and I'm exhausted.